src/StorageMover/StorageMover.Autorest/test/Start-AzStorageMoverJobDefinition.Recording.json

```diff
@@ -1,44 +0,0 @@
-{
```

Comment:
Do you have any context about why this two json files are removed? Is it because in previous PRs we deleted the test script for those commands by mistake?

Reply:
Hmm these two json files should be removed earlier but not now, but I'm not sure why these were not removed in previous PRs. The tests for Start/Stop-AzStorageMoverJobDefinition have been moved to Get-AzStorageMoverJobRun.Tests.ps1 previously.